scipy/stats/_mgc.py

```diff
@@ -96,7 +96,7 @@ def _euclidean_dist(x):
                               ['statistic', 'pvalue', 'mgc_dict'], [])
 
 
-@xp_capabilities(np_only=True)
+@xp_capabilities(out_of_scope=True)
```

Comment:
This was added with limited community support under the premise that the parent library, `mgcpy`, would be deprecated and the functionality needed a new home. But instead of being deprecated, `mgcpy` was rebranded as `hyppo`, which has a lot of related functionality. I think the community would be better served letting `hyppo` maintain this going forward.